src/Microsoft.Sbom.Common/Config/IConfiguration.cs

```diff
@@ -211,7 +211,7 @@ public interface IConfiguration
     /// <summary>
     /// The compliance standard to validate against.
     /// </summary>
-    public ConfigurationSetting<string> ComplianceStandard { get; set; }
+    ConfigurationSetting<ComplianceStandardType> ComplianceStandard { get; set; }
```

Comment:
`public` keyword is required for latest Visual Studio
```suggestion
    public ConfigurationSetting<ComplianceStandardType> ComplianceStandard { get; set; }
```